Corrections for user interface

chaehni opened the pull request:
This PR contains some corrections for the user interface:
- The 'link resend' page was changed to match the design  and behavior of the other pages
- After re-sending a verification email the input box is now emptied
- The link on the login page pointing to the 'link resend' page was re-placed

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/netsec-ethz/scion-coord/108)
<!-- Reviewable:end -->

mlegner commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> Only two minor comments.

---

Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

*[public/partials/resend.html, line 3 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-KyunAXH-crWWRRbffoZ:-KyunAXH-crWWRRbffo_:b-kryfga) ([raw file](https://github.com/netsec-ethz/scion-coord/blob/b3406d00727d2ef1199c2a1d875e570099778d0c/public/partials/resend.html#L3)):*
> ```HTML
> <div class="login-box">
>   <div class="login-logo">
>     <b>Oops, your account is not verified</b>
> ```

As users also see this page when they click on the "resend activation link" link, we could have a slightly more general headline here.

---

*[public/partials/resend.html, line 30 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-Kyun3xmIl4ndbVll6lB:-Kyun3xmIl4ndbVll6lC:b-5h8eex) ([raw file](https://github.com/netsec-ethz/scion-coord/blob/b3406d00727d2ef1199c2a1d875e570099778d0c/public/partials/resend.html#L30)):*
> ```HTML
> 
>     <br>
>     <a href="#/login" class="text-center">I'm already verfied</a>
> ```

We may want to write "I am", this would be consistent to the link on the registration page.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-:-KyunegZ-Tst706xsYZh:b-x2yei)*
<!-- Sent from Reviewable.io -->

ercanucan commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> (agree with @mlegner's comments)

---

Review status: 0 of 3 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-:-Kz4W6Jhs8ZBzpKYwo3J:bhoehgo)*
<!-- Sent from Reviewable.io -->

ercanucan commented:
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108)*
<!-- Sent from Reviewable.io -->

chaehni commented:
I updated the wording. I also added two more fixes for links overlapping on small screens and the bottom navbar cutting off content.

---

Review status: 1 of 4 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[public/partials/resend.html, line 3 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-KyunAXH-crWWRRbffoZ:-Kz5Y0pt9Qwk4veZA4Ix:b-896fix) ([raw file](https://github.com/netsec-ethz/scion-coord/blob/b3406d00727d2ef1199c2a1d875e570099778d0c/public/partials/resend.html#L3)):*
<details><summary><i>Previously, mlegner (Markus Legner) wrote…</i></summary><blockquote>

As users also see this page when they click on the "resend activation link" link, we could have a slightly more general headline here.
</blockquote></details>

Done.

---

*[public/partials/resend.html, line 30 at r1](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-Kyun3xmIl4ndbVll6lB:-Kz5Y3m2HW3Awu15mTvK:b-896fix) ([raw file](https://github.com/netsec-ethz/scion-coord/blob/b3406d00727d2ef1199c2a1d875e570099778d0c/public/partials/resend.html#L30)):*
<details><summary><i>Previously, mlegner (Markus Legner) wrote…</i></summary><blockquote>

We may want to write "I am", this would be consistent to the link on the registration page.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-:-Kz5ZYOwCORrK7r2xPNR:bfzudhn)*
<!-- Sent from Reviewable.io -->

ercanucan commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Review status: 1 of 4 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-:-Kz981LVTy9y5nByf5nx:bnfp4nl)*
<!-- Sent from Reviewable.io -->

ercanucan commented:
Reviewed 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108)*
<!-- Sent from Reviewable.io -->

mlegner commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> Please rebase.

---

Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/netsec-ethz/scion-coord/108#-:-Kz9A_f3RYA0Mp3f8pDh:b-e38at9)*
<!-- Sent from Reviewable.io -->

ercanucan commented:
merging.
